<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioWidget</class>
 <widget class="QWidget" name="AudioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="btn_mute">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbox_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="title">
    <string>function</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_task">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_prev">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>prev</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_volumn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_control">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_timecode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
